create-convex-hull-around-points/src/main/java/com/esri/arcgismaps/sample/createconvexhullaroundpoints/MainActivity.kt

```diff
@@ -143,45 +142,49 @@ class MainActivity : AppCompatActivity() {
         // recreate the graphics geometry representing the input points
         pointGraphic.geometry = Multipoint(inputPoints)
         // enable all the action buttons, since we have at least one point drawn
-        createButton.enable()
-        resetButton.enable()
+        createButton.isEnabled = true
+        resetButton.isEnabled = true
     }
 
     /**
      * Creates and draws a convex hull graphic on the map using [pointGeometry] points
      */
     private fun createConvexHull(pointGeometry: Geometry) {
         // normalize the geometry for panning beyond the meridian
-        // and check to proceed if the resulting geometry is not null
-        GeometryEngine.normalizeCentralMeridian(pointGeometry)?.let { normalizedPointGeometry ->
-            // create a convex hull from the points and proceed if it's not null
-            GeometryEngine.convexHull(normalizedPointGeometry)?.let { convexHullGeometry ->
-                // the convex hull's geometry may be a point or polyline if the number of
-                // points is less than 3, set its symbol accordingly
-                convexHullGraphic.symbol = when (convexHullGeometry) {
-                    is Point -> {
-                        // set symbol to use the pointSymbol
-                        pointSymbol
-                    }
-                    is Polyline -> {
-                        // set symbol to use the lineSymbol
-                        lineSymbol
-                    }
-                    is Polygon -> {
-                        // set symbol to use the fillSymbol
-                        fillSymbol
-                    }
-                    else -> {
-                        showError("Unknown geometry for convex hull")
-                        null
-                    }
-                }
-                // update the convex hull graphics geometry
-                convexHullGraphic.geometry = convexHullGeometry
-                // disable the create button until new input points are created
-                createButton.disable()
-            } ?: showError("Error creating convex hull")
+        val normalizedPointGeometry = GeometryEngine.normalizeCentralMeridian(pointGeometry)
+        // check to proceed if the resulting geometry is not null
+        normalizedPointGeometry ?: run {
+            // if it is null, show the error and return
+            showError("Error normalizing point geometry")
+            return
         }
```

Comment:
I would say if the null branch has multiple statements then it's better to just use an if statement
```kotlin
if (normalizedPointGeometry == null) { // ...
```
But both ways are fine, so I'm happy to approve